code/index.html

```diff
@@ -2,21 +2,98 @@
 <html>
   <head>
     <meta charset="utf-8" />
-    <title>Project 2: The News Site</title>
+    <title>HEALTH HACK</title>
     <link rel="stylesheet" href="./style.css" />
-    <!-- dont forget the meta tag for responsiveness -->
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+     
   </head>
   <body>
+
+     <!-- Header with logo, sitename and navbar -->
     <header>
-      <!-- Some kind of logo and navbar -->
+      <img src="HealthyHackLogo.gif" class="logo" alt="HealthHackLogo">
+      <h1>HEALTH HACK NEWS</h1>
+      <div class="links"></div>
+        <a href="#home" class="linkleft">Home</a>
+        <a href="#news" class="linkleft">News</a>
+        <a href="#contact" class="linkright">Contact</a>
+        <a href="#about" class="linkright">About</a>
+      <div>
     </header>
-
+      
+    <!-- Sections with news/cards-->
     <section class="big-news">
-      <!-- The big news content -->
+      <img src="cloudberry.jpg" alt="cloudberry">
+      <div class="big-news-text">
+       <h3>HACK SEROTONIN: EXPLORE THE CLOUDBERRY SEASON</h3>
+       <p>Now is the time to go hunt cloudberries, and while walking in nature also gain some serotonin. This year in Sweden we have a so called "cloudberry year" and you can find cloudberries almost everywhere in Sweden.</p>
+      </div>
     </section>
 
-    <section class="">
-      <!-- The other news  -->
+    <section class="other-news-cards">
+      <div class="card">
+        <img src="goodfood.jpg" alt="good-food" style="width:100%">
+        <div class="container">
```

Comment:
Consider naming this `text-container` to be even more descriptive?

Reply:
Ah okay :)